### ISSUE TITLE
Bug multiple textinput focused

### ISSUE THREAD

**Issue by uitlaber** [danvick/flutter_form_builder#683]
After edit textinput and click next textinput i have 2 focused input 

```
FormBuilder(
                      key: _formKey,
                      child: Column(
                        mainAxisAlignment: MainAxisAlignment.start,
                        crossAxisAlignment: CrossAxisAlignment.start,
                        children: [
                          FormBuilderImagePicker(
                              decoration: InputDecoration(
                                hintText: 'Фото нуждающегося',
                              ),
                              name: 'photo',
                              maxImages: 5,
                              bottomSheetPadding: EdgeInsets.all(40)),
                          SizedBox(
                            height: 20,
                          ),
                          FormBuilderTextField(
                            name: "fio",
                            decoration:
                                uiInputDecoration(hintText: 'ФИО нуждающегося'),
                            validator: FormBuilderValidators.compose([
                              FormBuilderValidators.required(context,
                                  errorText: 'Введите ФИО нуждающегося'),
                            ]),
                          ),
                          SizedBox(
                            height: 20,
                          ),
                          FormBuilderDropdown(
                            name: 'region',
                            decoration: uiInputDecoration(),
                            allowClear: true,
                            hint: Text('Город нуждающегося'),
                            items: List<DropdownMenuItem>.from(regions
                                    ?.map((region) => DropdownMenuItem(
                                        value: region.id,
                                        child: Text(region.name)))
                                    ?.toList() ??
                                []),
                          ),
                          SizedBox(
                            height: 20,
                          ),
                          FormBuilderTextField(
                            name: "address",
                            decoration: uiInputDecoration(
                                hintText: 'Адрес нуждающегося'),
                          ),
                          SizedBox(
                            height: 20,
                          ),
                          FormBuilderTextField(
                            name: "note",
                            maxLines: 5,
                            decoration: InputDecoration(
                                contentPadding: EdgeInsets.only(
                                    top: 20,
                                    bottom: 20,
                                    left: 20.0,
                                    right: 20.0),
                                border: new OutlineInputBorder(
                                    borderRadius: const BorderRadius.all(
                                      const Radius.circular(30.0),
                                    ),
                                    borderSide: new BorderSide(
                                        color: Colors.transparent)),
                                enabledBorder: new OutlineInputBorder(
                                    borderRadius: const BorderRadius.all(
                                      const Radius.circular(30.0),
                                    ),
                                    borderSide: new BorderSide(
                                        color: Colors.transparent)),
                                filled: true,
                                hintStyle:
                                    new TextStyle(color: Colors.grey[600]),
                                hintText: "Дополнительная информация  ",
                                fillColor: Colors.white),
                          ),
                          SizedBox(
                            height: 20,
                          ),
                          FormBuilderTextField(
                              name: "phone",
                              keyboardType: TextInputType.phone,
                              decoration: uiInputDecoration(
                                  hintText: '+7 (___) ___-__-__'),
                              validator: FormBuilderValidators.compose([
                                FormBuilderValidators.required(context,
                                    errorText: 'Введите ваш телефон'),
                                FormBuilderValidators.match(context,
                                    r'^\+7 \(([0-9]{3})\) ([0-9]{3})-([0-9]{2})-([0-9]{2})$',
                                    errorText: 'Неверный номер телефона')
                              ]),
                              inputFormatters: [phoneFormatter]),
                          SizedBox(
                            height: 20,
                          ),
                          ButtonTheme(
                            height: 50.0,
                            shape: RoundedRectangleBorder(
                              borderRadius: BorderRadius.circular(30.0),
                              // side: BorderSide(color: Colors.red)
                            ),
                            child: SizedBox(
                              width: double.infinity,
                              height: 50.0,
                              child: RaisedButton(
                                color: HexColor('#00748A'),
                                textColor: Colors.white,
                                onPressed: () => requestForm(),
                                elevation: 0,
                                child: Text(
                                  "Отправить в фонды ",
                                  style: TextStyle(
                                    fontSize: 18.0,
                                    fontWeight: FontWeight.bold,
                                    color: Colors.white,
                                  ),
                                ),
                              ),
                            ),
                          )
                        ],
                      )),
```

**Comment by sbx12** [comment]
One Solution I found was to create a Focus Node for each field.
Example 
class Your_Name_State extends State<Your_Name_Screen> {
final FocusNode _fioFocus = FocusNode();
final FocusNode _regionFocus = FocusNode();
final FocusNode _phoneFocus = FocusNode();


Then apply it to 
FormBuilderTextField(
  focusNode: _fioFocus ,
......
......
and do that for each form field even non text fields.
That worked for me and didn't have issues but this was my only solution.

If one the fields don't have the focus node assign to them. 
Then the issue will appear.

Hope this helps

For more help and where I found my solution
https://stackoverflow.com/questions/56038010/multiple-cursor-in-forms-textformfield-flutter
